src/uuid.js

```diff
@@ -0,0 +1,12 @@
+function S4() {
```

Comment:
S4?

Reply:
@kevinch Copy and paste da internet, porém o nome parece significar claramente String de tamanho 4, não é tão ruim até hehehe